### PR TITLE
[style] consistent nn. and nn.functional: part 4 `examples`

### PR DESCRIPTION
**Pull request by stas00**
This concludes the work on https://github.com/huggingface/transformers/issues/11600 with normalizing `examples` using fully automated:

```
# deal with torch.nn
perl -pi -e 's|^(\s*)import torch\n|$1from torch import nn\n$1import torch\n|' `grep -Ilr torch.nn examples`
find examples -type f -exec perl -X -pi -e 's{(?<!(from |import |[`#/]))torch\.nn\.}{nn.}g' {} \;
find examples -type f -exec perl -pi -e 's|import torch\.nn as nn|from torch import nn|g' {} \;

# deal with F
find examples -type f -exec perl -pi -e 's|from torch.nn import functional as F|from torch import nn|g' {} \;
find examples -type f -exec perl -pi -e 's|import torch.nn.functional as F|from torch import nn|g' {} \;
find examples -type f -exec perl -pi -e 's|(?<!\w)F\.|nn.functional.|g' {} \;
perl -pi -e 's|import torch||' examples/research_projects/pplm/pplm_classification_head.py

# leave legacy unmodified as we can't test is easily
git checkout examples/legacy

make fixup
```

@sgugger

### CODE REVIEW

**Review by sgugger — COMMENTED** [review]
You have changes in two png files here too, which is weird. Not sure if we really need to apply this to the research projects which are not actively maintained.

**Comment by stas00** [comment]
> You have changes in two png files here too, which is weird. 

Whoah! Found some secret stenography embeddings! :)

Probably got triggered by `s|(?<!\w)F\.|nn.functional.|g`

Thank you for noticing, @sgugger  - will fix it up!

> Not sure if we really need to apply this to the research projects which are not actively maintained.

at least - `wav2vec2` is. Do you want me to reset all but `wav2vec2` under research?

**Comment by sgugger** [comment]
No, it's easier to go for all of them in that case.
